app/controllers/api/v1/moves_actions.rb

```diff
@@ -14,6 +14,7 @@ def show_and_render
 
     def create_and_render
       move = Move.new(new_move_attributes)
+      move.determine_supplier
```

Comment:
I think now we can remove this extra step and add the `supplier` in the `new_move_attributes`.

Reply:
err.. no I don't think so. We need the move instance before we can call methods on it.

Reply:
ok, it makes sense, I am just a bit worried every time we add new lines in the controllers